src/query/__tests__/field.test.ts

```
@@ -0,0 +1,227 @@
+import { describe, it, expect } from 'vitest';
+import {
+  createStringField,
+  createNumberField,
+  createDropdownField,
+  createCheckboxField,
+  createDateField,
+  createUserField,
+  toString,
```

Biome (2.1.2)

[error] Do not shadow the global "toString" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)